### ISSUE TITLE
New position of added item?

### ISSUE THREAD

**Issue by fhanauer** [ManifestWebDesign/angular-gridster#453]
How can i retrieve the new `col` and `row` position right after adding item with `push(item)` ?
When i add item, the `col` and `row` are `undefined`, and at this time i need record the right positions on DB.
I tried the 'gridster-item-initialized' event, but i'm getting `null`.
Watching the `col` and `row` vars on item controller is the only aproach that worked , but seems to be like a hack.

An 'added-item' event with new the positions or even a `addItem()` function would be interesting in this case.

**Comment by lucasscarduelli** [comment]
Do you use angular to control itens of gridster?
When I add a new item by the angular the "row" and "col" attributes is set correctly.
Dont need to use any event to this... its only save in database what is in angular model.

**Comment by fhanauer** [comment]
Yes i'm using angular, but it takes some time to update the right positions internaly in the directive after the `push()`. Now, i am using a `timeout` right after the `push()` and then i can get the new positions.

**Comment by PythonDevOp** [comment]
One thing I noticed is that the gridster library sets the row/col values as NULL when adding to the gridster items array. When saving this to the database, how do you get around this fact?

**Comment by lucasscarduelli** [comment]
Do you use two-way-data-binding with the controller?
